examples/example-2/client/example-2.js

```diff
@@ -0,0 +1,95 @@
+// Create the Invoice constructor
+Invoice = ReactiveConstructor(function Invoice ( initData ) {
```

Comment:
Missing 'new' prefix when invoking a constructor.<br>'Invoice' is not defined.<br>'ReactiveConstructor' is not defined.